examples/poaps/backend/src/methods/fetch_single_poap.ts

```diff
@@ -0,0 +1,16 @@
+import { POAP, PoapsClient } from '@poap-xyz/poaps';
+import { PaginatedResult } from '@poap-xyz/utils';
+
+export const fetch_single_poap = async (client: PoapsClient): Promise<void> => {
+  try {
+    const data: PaginatedResult<POAP> = await client.fetch({
+      order: 'asc',
+      limit: 10,
+      offset: 0,
+      ids: [1],
+    });
+    console.log(data);
```

Comment:
Same here, would be nice to add some description and let know that even if is paginated, there will be zero or one POAP instances on the response.